### DISCUSSION TITLE
BDO Véleményezés

### DISCUSSION THREAD

**Discussion by BDO-Solutions** [nav-gov-hu/eVAT#16] [Kérdések és válaszok - Gépi interfész] [answered]
A BDO Magyarország adótanácsadói és IT Solutions szakemberei áttekintették a közzétett anyagokat. 
A koncepció alapvetően nagy előrelépés és segítség lenne az adózóknak, valamint a kínált kedvezmények vonzóbbá tehetik az átállást. Konkrét véleményt azonban a pontos specifikáció függvényében lehetséges alkotni. 

Az alábbi kérdések merültek fel a jelenleg közzétett koncepció kapcsán:
1.	Lévén, hogy az Online Számla is módosításra kerül, volna-e lehetőség a jelenlegi törvényeket figyelembe véve arra, hogy még a számlakiállítás megkezdése előtt, külön endpointon az ebben a koncepcióban felvázolthoz hasonló validációt végezzen az Online Számla? Ezzel ki lehetne küszöbölni az olyan formai hibákat, mint például a hibás irányítószám. 
2.	Az áfa bevallás önellenőrzése ebben az esetben egy külön bevallást jelentene? Technikailag ez hogyan lesz megoldva, illetve korlátozva? 
3.	Ahol nincs könyvelő rendszer (áfa regisztráltak), hogyan lesz megoldva a bevallás? Lesz valamilyen UI?
4.	Korábban felmerült, hogy az adóhatóság előállítja a bevallás tervezetet. Ha jól értelmezzük, akkor ennek lehetőségét nem vizsgálja továbbá az adóhatóság? 
5.	Amennyiben bevezetésre kerül az E-ÁFA, akkor annak használata opcionális lesz, vagy kiváltásra kerül az ÁNYK? 
6.	Van-e tervezett bevezetési időpont, illetve terveznek-e pilot projekt indítást? 
7.	Mikor várható a specifikáció tervezet közzététele?

Megtisztelő válaszukat előre is köszönjük!
Üdvözlettel:
BDO Magyarország csapata

**Comment by NTCA-tax** [comment, marked as answer]
Köszönjük a kérdéseiteket, amelyek közül több nálunk is nagyon sokszor felmerült. Fontos, hogy itt nem végleges dokumentációról beszélünk még, mivel a javaslatok alapján fogjuk véglegesíteni a scope-ot. Tehát ha valamelyik válasszal nem értenétek egyet, akkor azt javaslatként fogalmazzátok meg. Jelenleg még nagyon sok a nyitott kérdés nálunk is, és az év végéig el akarunk jutni oda, hogy ezeknek a száma nullához közelítsen...

1. Az Online Számla egy CTC megoldás, ott nem lehet előzetes validáció. Az eÁfa PTC, ezért tudjuk ezt ott megcsinálni. Teljesen más egy real-time rendszer működése, mint egy tranzakciótól alapvetően eltérő időpontban történő adatfeltöltés.
2. Az önellenőrzés külön XML feltöltését jelenti, és ott elvárás hasonlóan a jelenlegi 65-ös bevalláshoz a teljes adattartalom.
3. Szerintem a komplex ügyviteli rendszerrel rendelkezők sem feltétlen a saját ERP-ból fogják feltölteni az XML-t API-n keresztül. Ott is lesz egy köztes réteg, ahol összeállításra kerül az XML.  Az áfaregisztráltaknál is szerintem ki fognak alakulni azok a tool-ok, amelyek akár Excel bemenettel össze fogják tudni rakni az eÁfa M2M XML-t. Egy másik megoldás, hogy a kiajánlott áfabevallást fogja módosítani és azt adja be. Vagy ne vessük el az ÁNYK-n keresztüli beadást.
4. Nem vizsgáljuk, mert folyik a megvalósítása... A dokumentum 5. oldalának a tetején van utalás a bevallás kiajánlásra. Nem vetettük ezt el, folyik a munka hivatalon belül, de ez a dokumentum nem erről szól. 
5. A használata opcionális, a bevezetésével egyidejűleg nem állítjuk le az ÁNYK-t.
6. Vannak belső tervek a fejlesztési folyamatra, de annak publikálásához látnunk kell az M2M végleges scope-ját és a specifikációt.
7. Az idei évben biztosan nem, várhatóan jövő év elején.
